torch/jit/_trace.py

```diff
@@ -1046,7 +1046,7 @@ def check_unique(param):
 
         self.__dict__["_name"] = type(orig).__name__
         self.__dict__["_actual_script_module"] = script_module
-        for name in ("_parameters", "_buffers", "_modules"):
+        for name in ("_parameters", "_buffers", "_modules", "training"):
```

Comment:
Can you add a comment for why we need this?

Reply:
PyTorch v1.7.1 has a small bug in setting the `training` attribute for traced Module.
This bug is fixed in the master branch of PyTorch.
Please refer to [this](https://github.com/pytorch/pytorch/commit/ecfa7a27b8e6d121456c949688f2c01c6a6b7300).